docs/docs/configuration/hardware_acceleration.md

```diff
@@ -14,23 +14,29 @@ Ensure you increase the allocated RAM for your GPU to at least 128 (raspi-config
 ffmpeg:
   hwaccel_args: preset-rpi-64-h264
 ```
+##### H.264 and H.265 support
+**NOTICE**: Raspberry Pi 3 does not support hardware acceleration of H.265. H.265 can use a large amount of compute resources on the Raspberry Pi 3 and 4, in some cases it might be better to use H.264 streams.
```

Comment:
```suggestion

:::note

Raspberry Pi 3 does not support hardware acceleration of H.265. 

H.265 can use a large amount of compute resources on the Raspberry Pi 3 and 4, in some cases it might be better to use H.264 streams.

:::

```